src/appkit.cs

```diff
@@ -5915,39 +6056,33 @@ partial interface NSFont : NSSecureCoding, NSCopying {
 		bool IsFixedPitch { get; }
 
 		[Export ("boundingRectForGlyph:")]
+		[Deprecated (PlatformName.MacOSX, 10, 13, message: "Use CGGlyph APIs instead")]
 		CGRect BoundingRectForGlyph (uint /* NSGlyph = unsigned int */ aGlyph);
 
 		[Export ("advancementForGlyph:")]
+		[Deprecated (PlatformName.MacOSX, 10, 13, message: "Use CGGlyph APIs instead")]
 		CGSize AdvancementForGlyph (uint /* NSGlyph = unsigned int */ aGlyph);
 
-		// FIXME binding
```

Comment:
No, but they were deprecated. No reason to have FIXME bindings for deprecated bindings when we are binding the new hotness.